lib/importers/google/sessions/call_record.rb

```diff
@@ -0,0 +1,37 @@
+module Importers
+  module Google
+    module Sessions
+      class CallRecord
+        def initialize(cells)
+          @cells = cells
+        end
+
+        def params
+          {
+            delivery_partner: Partners::WEB_VISITS,
+            reporting_month: reporting_month,
+            transactions: transactions,
+            bookings: 0,
+            completions: 0
+          }
+        end
+
+        def unique_identifier
+          params.slice(:reporting_month, :delivery_partner)
+        end
+
+        def reporting_month
+          if @cells[0].to_i < 10
+            "#{@cells[1]}-0#{@cells[0]}"
+          else
+            "#{@cells[1]}-#{@cells[0]}"
+          end
+        end
```

Comment:
I find this a bit hard to understand (though maybe because I just don't have access to this test sheet yet). Maybe extracting some methods or some obvious variable names might help?